Scripts/Python/xYeeshaPages.py

```diff
@@ -211,87 +182,14 @@ def IDrawLinkPanel(self):
 
         #first hide them all
 #        PtDebugPrint("PageNumber = ", PageNumber.value)
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage01)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage02)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage03)).hide() 
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage04)).hide() 
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage05)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage06)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage07)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage08)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage09)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage10)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage12)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage13)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage14)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage15)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage16)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage17)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage18)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage19)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage20)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage21)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage22)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage23)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage24)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage25)).hide()
-        ptGUIControlButton(mydialog.getControlFromTag(kYeeshaPage26)).hide()
+        for x in kYeeshaPage:
+            try:
+                ptGUIControlButton(mydialog.getControlFromTag(x)).hide()
+            except:
+                continue
```

Comment:
```suggestion
                try:
                    ctrl = mydialog.getControlFromTag(x)
                except IndexError:
                    continue
                else:
                    ptGUIControlButton(ctrl).hide()
```

Reply:
This needed to be KeyError instead of IndexError.
Pushed the suggested changes from code review